README.md

```diff
@@ -976,7 +976,7 @@ See **[Ticketing](#ticketing)**
   * [Buddy GO](https://buddy.works/) - The Git and Continous Integration / Delivery Platform. `Fair` `Nodejs/Java`
   * [Concourse](https://concourse.ci/) - Concourse is a CI tool that treats pipelines as first class objects and containerizes every step along the way. ([Demo](https://ci.concourse.ci/), [Source Code](https://github.com/concourse/concourse)) `Apache` `Go`
   * [drone](https://drone.io/) - Drone is a Continuous Delivery platform built on Docker, written in Go. ([Source Code](https://github.com/drone/drone)) `Apache` `Go`
-  * [Factor.io](https://factor.io/) - It's like IFTTT (if-this-then-that) for Dev and DevOps. ([Source Code](https://github.com/factor-io/factor)) `MIT` `Ruby`
+  * [Factor](https://github.com/factor-io/factor) - It's like IFTTT (if-this-then-that) for Dev and DevOps. `MIT` `Ruby`
```

Comment:
If you browse to https://factor-io.github.io it redirects you to http://www.factor.io -- It's safe to assume that's probably what he's set as his custom domain for the GH pages repo. If we don't get a response in the next few days could you edit the PR to use this URL for the homepage @jgillich